vendor/codemirror/show-hint.js

```diff
@@ -0,0 +1,438 @@
+// CodeMirror, copyright (c) by Marijn Haverbeke and others
```

Comment:
Add link to the exact source file this was copied from

Reply:
Add `// Copied from https://github.com/codemirror/CodeMirror/blob/master/addon/hint/show-hint.js`